docs/src/pages/en/media-controller.md

```diff
@@ -42,92 +28,60 @@ Example (`autohide` disabled):
+
+Use this to turn off any built in or custom gestures, such as "click to toggle play/pause".
 
 Example (disabling gestures via `gestures-disabled`):
 
 ```html
 <media-controller gestures-disabled>
-  <video
-    slot="media"
-    src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"
-  ></video>
-  <media-control-bar>
-    <media-seek-backward-button></media-seek-backward-button>
-    <media-play-button></media-play-button>
-    <media-seek-forward-button></media-seek-forward-button>
-    <media-mute-button></media-mute-button>
-    <media-volume-range></media-volume-range>
-    <media-time-range></media-time-range>
-    <media-time-display></media-time-display>
-  </media-control-bar>
+  ...
 </media-controller>
 ```
 
-- `nohotkeys` - Use this to turn off _all_ keyboard shortcuts.
+### nohotkeys
+
+`nohotkeys` (boolean)
+
+Use this to turn off _all_ keyboard shortcuts.
 
 Example (hotkeys disabled):
 
 ```html
 <media-controller nohotkeys>
-  <video
-    slot="media"
-    src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"
-  ></video>
-  <media-control-bar>
-    <media-seek-backward-button></media-seek-backward-button>
-    <media-play-button></media-play-button>
-    <media-seek-forward-button></media-seek-forward-button>
-    <media-mute-button></media-mute-button>
-    <media-volume-range></media-volume-range>
-    <media-time-range></media-time-range>
-    <media-time-display></media-time-display>
-  </media-control-bar>
+  ...
 </media-controller>
 ```
 
-- `hotkeys` - Use this to turn off certain hotkeys.
+### hotkeys
+
+`hotkeys` (string of multiple values)
 
-If both `hotkeys` and `nohotkeys` are added to the MediaController, all hotkeys will be disabled.
+Use this to turn off certain hotkeys. If both `hotkeys` and `nohotkeys` are added to the MediaController, all hotkeys will be disabled.
 
 Example (disallow seeking shortcuts):
 
 ```html
 <media-controller hotkeys="noarrowleft noarrowright">
-  <video
-    slot="media"
-    src="https://stream.mux.com/DS00Spx1CV902MCtPj5WknGlR102V5HFkDe/high.mp4"
-  ></video>
-  <media-control-bar>
-    <media-seek-backward-button></media-seek-backward-button>
-    <media-play-button></media-play-button>
-    <media-seek-forward-button></media-seek-forward-button>
-    <media-mute-button></media-mute-button>
-    <media-volume-range></media-volume-range>
-    <media-time-range></media-time-range>
-    <media-time-display></media-time-display>
-  </media-control-bar>
+  ...
 </media-controller>
 ```
+
+### default-stream-type
+
+`default-stream-type` (values: `live`, `on-demand`)
+
+Media controller can't know the stream type of the media until the media is loaded. Setting `default-stream-type` can prevent UI changes happening between when the player is loaded and when the media is loaded. This may happen when a player is built to support both stream types, and then is used to play a stream type that is different from the player's default.
```

Comment:
suggestion(non-blocking):
Since we're at least kind of introducing a new concept "stream type," consider replacing 
> Media controller can't know the stream type of the media until the media is loaded.
with
> Media controller can't know if the media will be live or on demand until after the media is loaded.
or similar

Reply:
suggestion(non-blocking): Link to new stream-type docs page somewhere here "For more info on stream types, see"